cli/mender-artifact/artifacts.go

```diff
@@ -104,7 +104,7 @@ func unpackArtifact(name string) (string, error) {
 	aReader := areader.NewReader(f)
 	rootfs := handlers.NewRootfsInstaller()
 
-	tmp, err := ioutil.TempFile(filepath.Dir(name), "mender-artifact")
```

Comment:
Why have you changed this? Now the temporary file will be created in `/tmp` instead of the same location where the artifact is. I think the reason for storing it in the same location was that you might not have access to `/tmp` (I am not sure if this was the case for Travis), but you most probably can create some temporary files in your work directory.